release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -72,6 +72,18 @@ This release adds improvements related to the following components and concepts.
 [id="ocp-4-14-networking"]
 === Networking
 
+[id="ocp-4-14-networking-kernal-network-pinning"]
+==== Dynamic use of non-reserved CPUs for networking workloads
+
+With this release, networking workloads in performance-tuned clusters with a CPU manager policy set to `static` can dynamically use non-reserved CPUs by default. 
```

Comment:
Ok, this needs a bit of rewrite. First it is not the workloads that will get access to the cpus, it is OVS (openvswitch). The workloads using OVS networking will benefit from it.

Basically replace Networking workloads with openvswitch or "OVN networking stack" and update the rest accordingly.

Reply:
@MarSik - thanks Martin, I updated the text. Hope this works.